src/registry_schemas/example_data/schema_data.py

```diff
@@ -1217,6 +1217,17 @@
     'courtOrder': COURT_ORDER
 }
 
+CONTINUATION_OUT = {
+    'details': 'A note to explain the continue out',
+    'continuationOutDate': '2023-05-01',
+    'foreignJurisdiction': {
```

Comment:
Maybe not "foreign" in case it's a province or "federal"? (I consider "foreign" to be another country.)

Maybe just "jurisdiction"?

Reply:
We already use a `foreign_` prefix for a bunch of columns in the `businesses` table for some of the work we did for continuation in.   I think we should keep some kind of prefix even if it's not `foreign_`

How about `external_jurisdiction`?

![image](https://github.com/bcgov/business-schemas/assets/6685223/4914e79e-0bcd-4f2c-a6db-093f9d0790f0)

Reply:
What does "foreign" refer to in a Continuation In?

If we're talking about the same thing then I agree that keep it similar/same is better.

Reply:
It's the opposite of continuation out I believe.  i.e. a business is wanting to incorporate in BC but is coming from another jurisdiction.  I'm assuming jurisdiction in this case is the same as continuation out.

Reply:
OK. I was trying to get commonality between them. No worries. Resolving this conversation.